content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -125,7 +125,7 @@ If your configuration is not using the latest version it is **recommended** that
 the [kubeadm config migrate](/docs/reference/setup-tools/kubeadm/kubeadm-config/) command.
 
 For more information on the fields and usage of the configuration you can navigate to our API reference
-page and pick a version from [the list](https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm#pkg-subdirectories).
+page and pick a version from [the list](https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm#section-subdirectories).
```

Comment:
```suggestion
page and pick a version from [the list](https://pkg.go.dev/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm#section-directories).
```


it's https://pkg.go.dev/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm#section-directories like you said on https://github.com/golang/go/issues/45210

please keep the commits squashed to 1.

/sig cluster-lifecycle

Reply:
seems it's too late for today... Will fix this tomorrow ...

Reply:
I just approved that changes via GitHub. Is there no way to squash via the GUI like it is in GitLab?

Reply:
there is a bot command of sorts, the appovers know about.
one caveat is that it nukes GPG signatures on commits.